### ISSUE TITLE
Display server-side errors in Certification v2 [2]

### ISSUE THREAD

**Issue by amprokop** [department-of-veterans-affairs/caseflow#1716]
Make sure appropriate error pages are rendered for server-side errors.

- if there's an error saving to the DB
- if there's an error certifying the appeal to VACOLS or uploading the docs to VBMS

This is blocked by #1313 and #1642

**Comment by amprokop** [comment]
In the PR post screenshots so @shellicious can approve.

**Comment by amprokop** [comment]
We should mimic the pages currently in Certification.

**Comment by kierachell** [comment]
**PASSED**
## Saving to DB error:
```
ypeError (no implicit conversion of nil into String):                                                         app/services/form8_pdf_service.rb:150:in `+'                                                                                                                                                                                               app/services/form8_pdf_service.rb:150:in `block in pdf_values_for'                                                                                                                                                                         app/services/form8_pdf_service.rb:128:in `each'                                                                                                                                                                                            app/services/form8_pdf_service.rb:128:in `each_with_object'
  app/services/form8_pdf_service.rb:128:in `pdf_values_for'                                                                                                                                                                                  app/services/form8_pdf_service.rb:166:in `save_pdf_for!'                                                                                                                                                                                   app/models/form8.rb:46:in `save_pdf!'                                                                                                                                                                                                      app/controllers/certifications_controller.rb:58:in `certify_v2'                                                                                                                                                                            app/middleware/metrics_collector.rb:14:in `call'                                                                                                                                                                                           app/middleware/metrics_auth.rb:10:in `call'
```
<img width="1108" alt="screen shot 2017-05-19 at 12 46 52 pm" src="https://cloud.githubusercontent.com/assets/18075411/26258140/632aca3c-3c91-11e7-8339-2c770548dd88.png">

## VACOLS error
```
Appeal Load (1.6ms)  SELECT  "appeals".* FROM "appeals" WHERE "appeals"."vacols_id" = $1 LIMIT 1  [["vacols_id", "2765716"]]
dsva-appeals-certification-uat/opt/caseflow-certification/src/log/caseflow-certification.out i-09149b535e17c14d2-172.30.71.93 [appeals.cf.uat.ds.va.gov] [0f952b8c-bbd7-4683-8706-4aa795b64f44] [172.30.71.93] [CASEFLOW_397   caseflow@va.gov] [2017-05-19 12:53:35 -0400] STARTED VACOLS: load_vacols_data 2765716
dsva-appeals-certification-uat/opt/caseflow-certification/src/log/caseflow-certification.out i-09149b535e17c14d2-172.30.71.93 [appeals.cf.uat.ds.va.gov] [0f952b8c-bbd7-4683-8706-4aa795b64f44] [172.30.71.93] [CASEFLOW_397   caseflow@va.gov] [2017-05-19 12:53:35 -0400]   VACOLS::Case Load (102.8ms)  SELECT  "VACOLS"."BRIEFF".* FROM "VACOLS"."BRIEFF" WHERE "VACOLS"."BRIEFF"."BFKEY" = :a1 AND ROWNUM <= 1   [["bfkey", "2765716"]]
dsva-appeals-certification-uat/opt/caseflow-certification/src/log/caseflow-certification.out i-09149b535e17c14d2-172.30.71.93 [appeals.cf.uat.ds.va.gov] [0f952b8c-bbd7-4683-8706-4aa795b64f44] [172.30.71.93] [CASEFLOW_397   caseflow@va.gov] [2017-05-19 12:53:35 -0400] Completed 404 Not Found in 1337ms (Views: 0.2ms | ActiveRecord: 113.6ms)
dsva-appeals-certification-uat/opt/caseflow-certification/src/log/caseflow-certification.out i-09149b535e17c14d2-172.30.71.93 172.30.69.231, 172.30.70.242 - - [19/May/2017:12:53:35 -0400] "POST /certifications/2765716/certify_v2 HTTP/1.0" 404 - 1.3413
```
<img width="1108" alt="screen shot 2017-05-19 at 12 46 52 pm" src="https://cloud.githubusercontent.com/assets/18075411/26258344/56fb4c2c-3c92-11e7-910d-1e2d21858150.png">

Doesn't say that it's a VACOLS error, but no one cares :D

**Comment by nemodjuric** [comment]
There should be no header when error is displayed.

**Comment by nemodjuric** [comment]
New issue is created - https://github.com/department-of-veterans-affairs/caseflow/issues/2077
Closing this one.
